styles/google-photos/catppuccin.user.less

```
@@ -54,7 +69,9 @@
     @crust: @catppuccin[@@flavor][@crust];
     @accent: @catppuccin[@@flavor][@@accentColor];
 
-    color-scheme: if(@flavor = latte, light, dark);
+    @inverseTheme: if(@flavor = latte, mocha, latte);
+    @inverseAccent: @catppuccin[@@inverseTheme][@@accentColor];
```

Let's use kebab-case for Less variables please (helps distinguish from the metadata vars which are camelCase).
```suggestion
    @inverse-theme: if(@flavor = latte, mocha, latte);
    @inverse-accent: @catppuccin[@@inverse-theme][@@accentColor];
```